RecoBTag/Combined/python/hiRun2LegacyCSVv2Computer_cfi.py

```diff
@@ -0,0 +1,27 @@
+import FWCore.ParameterSet.Config as cms
+import RecoBTag.SecondaryVertex.candidateCombinedSecondaryVertexV2Computer_cfi as sl_cfg 
```

Comment:
can you rename sl_cfg to sv_cfg. The sl stands for soft lepton which is used in the c-tagger, but here you are just using the SV+track info, so the consistent naming with btv code would be sv